src/CreeDictionary/CreeDictionary/views.py

```diff
@@ -346,46 +345,24 @@ def should_include_auto_definitions(request):
     return request.user.is_authenticated
 
 
-def paradigm_for(
-    wordform: Wordform, paradigm_size: ParadigmSize | str
-) -> Optional[Paradigm]:
+def paradigm_for(wordform: Wordform, paradigm_size: str) -> Optional[Paradigm]:
     """
     Returns a paradigm for the given wordform at the desired size.
 
-    If a paradigm cannot be found, an empty list is returned
+    If a paradigm cannot be found, None is returned
     """
-    # TODO: make this use the new-style ParadigmManager exclusively
 
     manager = default_paradigm_manager()
 
-    if isinstance(paradigm_size, ParadigmSize):
-        size = convert_crkeng_paradigm_size_to_size(paradigm_size)
-    else:
-        size = paradigm_size
+    size = paradigm_size
```

Comment:
Is there an issue with changing the argument name of this function to get rid of this alias?

Reply:
No problem, just didn’t notice that.